website/docs/src/content/cloud/observability.mdx

```diff
@@ -0,0 +1,269 @@
+---
+title: Observability & tracing
+description: Monitor and debug your GenSX workflows
+---
+
+# Observability & tracing
+
+GenSX provides observability tools that make it easy to understand, debug, and optimize your workflows. Every component execution is automatically traced, giving you full visibility into what's happening inside your LLM workflows. You can view traces in realtime as workflows execute, and view historical traces to debug production issues like hallucinations.
```

Comment:
Should we add a "when logged in" caveat?

Reply:
eh, this section is under cloud so I this should be implicitly understood.